PhysicsTools/PatAlgos/python/producersLayer1/muonProducer_cfi.py

```diff
@@ -132,7 +133,7 @@
 
 
 
-
+eras.run2_miniAOD_80XLegacy.toModify(patMuons, mvaTrainingFile=cms.FileInPath("RecoMuon/MuonIdentification/data/MVA_Legacy_deepFlav_together_mu_2016_BDTG.weights.xml"))
```

Comment:
please import just ```run2_miniAOD_80XLegacy``` and apply toModify on it directly instead of ```eras.```.

Also, to avoid type mismatch and typos, please use ```mvaTrainingFile= "RecoMuon/MuonIdentification/data/MVA_Legacy_deepFlav_together_mu_2016_BDTG.weights.xml"```
without specifying a type